upgrade/upgrade.md

```diff
@@ -0,0 +1,6 @@
+# Upgrading Bold Reports to latest version
+
+This section explains how to upgrade Bold Reports to latest version in your Kubernetes cluster. You can refer to the features and enhancements from this [Release Notes](https://www.boldreports.com/release-history/embedded-reporting).
+
+* [Upgrade Bold Reports to latest version](/upgrade/5-2_upgrade.md)
+* [Upgrade Bold Reports to latest version if your current version is below 5.1](/upgrade/5-1_upgrade.md)
```

Comment:
Upgrade Bold Reports from older version (below v5.1) to latest

Reply:
changed